site/app/templates/CreateNewAccount.twig

```diff
@@ -0,0 +1,88 @@
+            <div class="hidden-helper" id="userid-helper">
+                UserID Requirements:<br/>
+                {% if user_id_reqs["all"] == true %}
+                * Must be greater than {{ user_id_reqs["min_length"] }} 
+                    characters, and less than
+                    {{ user_id_reqs["max_length"] }}
+                    characters in length 
+                {% elseif user_id_reqs['require_name'] == true %}
+                    * Must be
+                    {{ name_reqs['given_first'] ? 
+                    (name_reqs['given_name'] ~ ' characters of your given name followed by ') : 
+                    (name_reqs['family_name'] ~ ' characters of your family name followed by' )}}
+                    {{ name_reqs['given_first'] ? 
+                    (name_reqs['family_name'] ~ ' characters of your family name') : 
+                    (name_reqs['given_name'] ~ ' characters of your given name' )}}
+                {% elseif user_id_reqs['require_email'] == true %}
+                    {% if user_id_reqs['whole_email'] %} 
+
+                    {% endif %}
+                {% endif %}
+            </div>
+            <input type="text" name="user_id" data-testid="user-id" required />
+        </div>
+        <div class="display-box">
+            Given Name
+            <input type="text" name="given_name" data-testid="given-name" required />
+        </div>
+        <label>
+            Family Name
+            <input type="text" name="family_name" data-testid="family-name" required />
+        </label>
+        <div class="display-box">
+           <p style="display:block">Password <button class="requirements-button" type="button" onclick="showRequirements(this.id)" id="password">Show Requirements</button></p>
+            <div class="hidden-helper" id="password-helper">
+                Password Requires at least:<br/>
+                1 Capital Letter<br/>
+                1 Lowercase Letter<br/>
+                1 Number<br/>
+                1 Special Character<br/>
+                12 Characters<br/>
+            </div>
+            <input type="password" name="password" data-testid="password" required />
+        </div>
+        <label>
+            Re-Enter Password
+            <input type="password" name="confirm_password" data-testid="confirm-password" required />
+        </label>
+        <input type="submit" name="signup" value="Sign Up" class="btn btn-primary" data-testid="sign-up-button" />
+    </form>
+</div>
+
+
+<script>
+
+function showRequirements(id_string) {
+    id_string = id_string + '-helper';
+    if (document.getElementById(id_string).style.display === '' || document.getElementById(id_string).style.display === 'none') {
+        document.getElementById(id_string).style.display = 'block';
+    } else {
+        document.getElementById(id_string).style.display = 'none';
+    }
```

Comment:
You can use jQuery's [`.toggle()`](https://api.jquery.com/toggle/) method for this.